Minecraft_Quick_Homes/Minecraft_Quick_Homes.ahk

```diff
@@ -345,15 +386,394 @@ Wait_Until_Minecraft_Registers_Slash()
+                this.what_is_the_difference__in_two_indexes()
+            }
+            
+            ;msgbox % this.last_and_new_minecraft_title[1]
+            
+
+            ; msgbox % this.window_title_change
+            return this.window_title_change
+        }
+    }
+    
+    store_section_pos_after_auto_Switch(Active_Window_Title)
+    {
+        global Stored__home_section_pos
+        Stored__home_section_pos := switch_minecraft_header_sections.Switch_Section_by_category(0, Active_Window_Title)
+
+    }
+
+    Auto_Switch_Sections__by_Minecraft_Title()
+    {
+        IniRead, Auto_Switch_Sections_by_Minecraft_Title, HomeStorage.ini, config, Auto_Switch_Sections_by_Minecraft_Title
+
+        
+
+        window_title_change := This.compare.Has_Window_title_changed()
+
+        if Auto_Switch_Sections_by_Minecraft_Title = 1
+        {
+            if window_title_change = 1
+            {
+                ; Get the active window title
+                Active_Window_Title := minecraft_version_sections_ReadWrite.Get_Active_Window_Title()
+                ;msgbox % Active_Window_Title
+
+                ; Save Stored__home_section_pos
+                this.store_section_pos_after_auto_Switch(Active_Window_Title)            
+                ; Calculate home based on window title. (Use minecraft_section_switch)
+                home_section := this.Switch_Section_by_category(0, Active_Window_Title)
+            }
+        }
+
+        ; If the setting is not turned on, then this options should return "" by default. 
+        return home_section
+    }
+
+
+}
+
+get_section_home_name__from_section_pos(section_pos)
+{
+    home_sections := minecraft_version_sections_ReadWrite.Parse_Sections_Homes_Into_Array()
+    home_name := home_sections[0][section_pos]
+
+    return home_name
+}
+
+; get_section_name is meant to be used with Switch_Section_by_category(). If you need to use a message, enable the second parameter.
+Show_tooltip_while__section_combo_held__(get_section_pos__or__message_input, message_option)
+{
+
+    home_name := get_section_home_name__from_section_pos(get_section_pos__or__message_input)
+
+    
+
+    while GetKeyState("Ctrl", "P") || GetKeyState("Alt", "P")
+    {
+        if message_option = 0
+            ToolTip, %home_name%
+
+        if message_option = 1
+            ToolTip, %get_section_pos__or__message_input%
+    }
+
+    ToolTip
+
+
+    return
+}
+
+
+determine_home_name()
+{
+    global Stored__home_section_pos
+    ; This looks up the current Stored__home_section_pos. But if switch_minecraft_header_sections.Auto_Switch_Sections__by_Minecraft_Title() was on, it should run under this command. 
+    Home_Name_Default := get_section_home_name__from_section_pos(Stored__home_section_pos)
+    Home_Name_2 := get_section_home_name__from_section_pos(switch_minecraft_header_sections.Auto_Switch_Sections__by_Minecraft_Title())
+
+    ; If the Window title is not named, don't use it if the setting is turned on. 
```

Comment:
> If the Window title is not named, don't use it if the setting is turned on.

If the Window title is not named (aka `windowtitle := ""`), don't use it if the setting is turned on.

Reply:
Issue resolved. Added clarification.